### PR TITLE
fix: Use HA URL for chartmuseum

### PR DESCRIPTION
**Pull request by pmuir**
#### Submitter checklist

- [X] Change is code complete and matches issue description.
- [X] Change is covered by existing or new tests.

#### Description
Use HA URL for chartmuseum

#### Which issue this PR fixes

fixes #

### CODE REVIEW

File: pkg/jx/cmd/init.go [hunk 1]
```diff
@@ -73,7 +73,7 @@ const (
 	// INGRESS_SERVICE_NAME service name for ingress controller
 	INGRESS_SERVICE_NAME = "jxing-nginx-ingress-controller"
 	// DEFAULT_CHARTMUSEUM_URL default URL for Jenkins X ChartMuseum
-	DEFAULT_CHARTMUSEUM_URL = "https://chartmuseum.jx.cd.jenkins-x.io"
+	DEFAULT_CHARTMUSEUM_URL = "http://chartmuseum.jenkins-x.io"
```
Comment (agentgonzo) [inline pkg/jx/cmd/init.go]: why not https?

Comment (pmuir) [reply pkg/jx/cmd/init.go]: It's a GCS bucket and it's quite hard to add a cert to a GCS bucket (/cc @i0n). It would be nice though, so create an issue?

Comment (pmuir) [reply pkg/jx/cmd/init.go]: Well to be more direct, we don't have a valid cert right now :-)

Comment (agentgonzo) [reply pkg/jx/cmd/init.go]: We should probably get one ;-). How do we get HTTPS certs? letsencrypt or another provider?

**Pull request merged**